eva-lib/src/test/resources/eva.properties

```diff
@@ -6,7 +6,7 @@ eva.mongo.auth.db=testdb
 eva.mongo.read-preference=
 
 eva.mongo.collections.variants=vars
-eva.mongo.collections.files=files
+eva.mongo.collections.files=files_1_2
```

Comment:
It should be possible to make this work using something a different name from "files_1_2", right? If so, it would be good to keep it different to avoid any confusion.